MN.L10n.BuildTasks/Program.cs

```diff
@@ -106,12 +106,16 @@ static int Main(string[] args)
 						.ToList();
 					}
                     else
-                    { 
-						fileList.AddRange(
-						    Directory.EnumerateFiles(solutionDir, "*.*", SearchOption.AllDirectories)
-                            .Where(f => config.IncludePatterns.Any(p => f.ToLower().Contains(p.ToLower())))
-                            .Where(f => !defaultIgnorePaths.Any(ign => f.ToLower().Contains(ign)))
-						);
+                    {
+						var fileListWithIgnores = Directory.EnumerateFiles(solutionDir, "*.*", SearchOption.AllDirectories)
+							.Where(f => !defaultIgnorePaths.Any(ign => f.ToLower().Contains(ign)));
+
+						var filesListWithoutIgnores = Directory.EnumerateFiles(solutionDir, "*.*", SearchOption.AllDirectories)
```

Comment:
Bordew inte denna heta 
filesLIstWithIncludes för du tar ju inte allt utan explicit de som är includerade :)

Reply:
Yes, it should.. I can fix it after I've merged. ^^;